citestwheel/citestwheel.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -eoxu pipefail
+
+export RAPIDS_PY_WHEEL_NAME="${RAPIDS_PY_WHEEL_NAME:-}"
+export RAPIDS_WHEEL_VER_OVERRIDE="${RAPIDS_WHEEL_VER_OVERRIDE:-}"
+export CIBW_TEST_EXTRAS="${CIBW_TEST_EXTRAS:-}"
+export CIBW_TEST_REQUIRES="${CIBW_TEST_REQUIRES:-}"
+export CIBW_TEST_COMMAND="${CIBW_TEST_COMMAND:-}"
+export RAPIDS_BEFORE_TEST_COMMANDS_AMD64="${RAPIDS_BEFORE_TEST_COMMANDS_AMD64:-}"
+export RAPIDS_BEFORE_TEST_COMMANDS_ARM64="${RAPIDS_BEFORE_TEST_COMMANDS_ARM64:-}"
+
+rm -rf ./dist
+mkdir -p ./dist
+
+first='yes'
+arch=$(uname -m)
+
+for pyver in ${RAPIDS_PY_VERSION}; do
+        deactivate || true
+
+        pytestenv="cp${pyver//./}-cp${pyver//./}"
+
+        /opt/python/$pytestenv/bin/python -m venv /cibw-test-venv-${pyver}
+        . /cibw-test-venv-${pyver}/bin/activate
+
+        curl -sS https://bootstrap.pypa.io/get-pip.py | python3
+
+        if [ "${arch}" == "x86_64" ]; then
+                sh -c "${RAPIDS_BEFORE_TEST_COMMANDS_AMD64}"
+        elif [ "${arch}" == "aarch64" ]; then
+                sh -c "${RAPIDS_BEFORE_TEST_COMMANDS_ARM64}"
+        fi
+
+        if [ "${first}" == "yes" ]; then
```

Comment:
Consider a light-weight 'wheel ci' container that has the initial set of tools:
* pipx
* cibuildwheel
* awscli

Things that are needed for pure-wheels, manylinux-wheels, etc.